js/src/admin/AdminApplication.js

```diff
@@ -64,4 +70,14 @@ export default class AdminApplication extends Application {
 
     return required;
   }
+
+  setDefaultExtensionPages() {
```

Comment:
I'm still a bit uncertain about registering separate routes for each discussion, but as we discussed yesterday, that might be the best of our options for extensibility.